package.json

```diff
@@ -165,7 +165,7 @@
     "react-timer-mixin": "^0.13.2",
     "react-transform-hmr": "^1.0.4",
     "rebound": "^0.0.13",
-    "regenerator": "^0.8.36",
+    "regenerator-runtime": "^0.9.5",
```

Comment:
No worries, @benjamn, these kind of breaks happen from time to time, thanks a lot for considering a more robust versioning in the future.
Very much appreciated 👍